apache/airflow/ExposedUI/README.md

```diff
@@ -0,0 +1,65 @@
+# vulnerble airflow instance
+
+```bash
+# on ubuntu
+sudo apt install python3
+sudo apt install python3-venv
+sudo apt install python3-pip
+mkdir airflowWorkSpace
+cd airflowWorkSpace
+python3 -m venv .venv
+source .venv/bin/activate
+pip install apache-airflow
+
+mkdir config
+export AIRFLOW_HOME="$(pwd)/config"
+export AIRFLOW__CORE__TEST_CONNECTION=Enabled
+
+# create and edit config/webserver_config.py file
+touch config/webserver_config.py
+cat << EOF >> config/webserver_config.py 
+from __future__ import annotations
+import os
+from flask_appbuilder.const import AUTH_DB
+basedir = os.path.abspath(os.path.dirname(__file__))
+WTF_CSRF_ENABLED = True
+WTF_CSRF_TIME_LIMIT = None
+AUTH_TYPE = AUTH_DB
+AUTH_ROLE_PUBLIC = 'Admin'
+EOF
+```
+
+* run `airflow standalone` and then go to http://127.0.0.1:8080/home we can see no auth is needed for anything you want to do.
+* go to http://127.0.0.1:8080/connection/add and select HTTP connection type, you can see the test button is enabled so we can test out-of-band calls to verify the exposed Dashboard/UI.
+* ref: https://airflow.apache.org/docs/apache-airflow-providers-fab/stable/auth-manager/webserver-authentication.html#webserver-authentication
+
+# safe airflow instance
+In a safe setup unlike the unsafe setup, we remove `AUTH_ROLE_PUBLIC = 'Admin'` from `config/webserver_config.py file`  and Also we don't export the `AIRFLOW__CORE__TEST_CONNECTION=Enabled` environment variable.
+
+```bash
+# on ubuntu
+sudo apt install python3
+sudo apt install python3-venv
+sudo apt install python3-pip
+mkdir airflowWorkSpaceSafe
+cd airflowWorkSpaceSafe
+python3 -m venv .venv
+source .venv/bin/activate
+pip install apache-airflow
+
+mkdir config
+export AIRFLOW_HOME="$(pwd)/config"
+
+# create and edit config/webserver_config.py file
+touch config/webserver_config.py
+cat << EOF >> config/webserver_config.py 
+from __future__ import annotations
+import os
+from flask_appbuilder.const import AUTH_DB
+basedir = os.path.abspath(os.path.dirname(__file__))
+WTF_CSRF_ENABLED = True
+WTF_CSRF_TIME_LIMIT = None
+AUTH_TYPE = AUTH_DB
+EOF
+```
+* now run `airflow standalone` and then go to http://127.0.0.1:8080/home, you need to authenticate so that anonymous access is not enabled.
```

Comment:
I would change the content with this:

1.  Run `airflow standalone`. You can now navigate to http://localhost:8080/home. 
2. Notice that, this time, you need to be authenticated to hit the endpoint. As a result of this, you will be redirect to the `/login` page.